packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -0,0 +1,709 @@
+---
+title: Authenticator Admin
+category: management
+---
+
+# Authenticator Admin API
+
+The Authenticator Admin API enables an Org Administrator to configure which Authenticators are available to end users to be used for signing into applications.
+
+End users would be required to use one or more Authenticators depending on the security requirements of the application sign-on policy.
+
+The Okta Identity Engine currently supports Authenticators for the following factors:
+
+**Knowledge-based:**
+* Password
+* Security Question
+
+**Possession-based:**
+* Phone (SMS, Voice Call)
+* Email
+* WebAuthn
+
+<!--
+## Get started
+
+Explore the Authenticator Admin API: [![Run in Postman](https://run.pstmn.io/button.svg)](%linkhere%). % How to generate this link: https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/227648499/How+to+publish+Postman+collections %
+
+-->
+
+## Authenticator Admin Operations
+
+The Authenticator Admin API has the following CRUD operations:
+
+* [List Authenticators](#list-authenticators)
+* [Get an Authenticator by id](#get-an-authenticator-by-id)
+* [Activate an Authenticator](#activate-an-authenticator)
+* [Deactivate Authenticator](#deactivate-an-authenticator)
+
+### List Authenticators
+
+<ApiOperation method="get" url="/api/v1/authenticators" />
+
+Creates a Authenticator object.
```

Comment:
Not sure how we got this line which is talking about creating (instead of listing).